### DISCUSSION TITLE
Couldn't determine the version of the native part of Reanimated

### DISCUSSION THREAD

**Discussion by Hinterberger-Thomas** [software-mansion/react-native-reanimated#5601] [Q&A]
I upgaded expo to version 50 today and after that react-native-reanmiated give me the error  "Couldn't determine the version of the native part of Reanimated". I allready looked at the troubleshoot page, but I have rebuild the app, wiped the virtual device, cleared the cache and nothing seems to work. Not sure if I am missing something or if this is just a bug with the new SDK. I also checked if the version of reanimted and the expo sdk are the same and they are: 3.6.1

**Comment by MatiPl01** [comment]
Could you try updating reanimated to `3.6.2`? This version was released yesterday and it should fix the issue.

**Comment by MatiPl01** [comment]
I think that this discussion can be safely closed. The issue is related to the old version of reanimated and is already fixed.

If you still have questions, feel free to re-open the discussion or start a new one.
